lms/djangoapps/courseware/views.py

```diff
@@ -1378,6 +1378,11 @@ def process_analytics_answer_dist(data, question_types_by_part, num_options_by_p
     # Count rows returned for each part for integrity check
     num_rows_by_part = {}
 
+    # List of part_ids to check for inconsistencies.
+    part_id_set = set([])
+    for _index, data_dict in enumerate(data):
```

Comment:
It looks like _index isn't used in the loop body, so I don't think you need the call to enumerate.